e2e_test.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main_test
+
+import (
+	"bytes"
+	"errors"
+	"flag"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+var update = flag.Bool("update", false, "update golden files")
+
+func TestCsvCommandE2E(t *testing.T) {
+	var tests = []struct {
+		workdir string
+	}{
+		{workdir: "testdata/modules/hello01"},
+		{workdir: "testdata/modules/cli02"},
+		{workdir: "testdata/modules/vendored03"},
+		{workdir: "testdata/modules/replace04"},
+	}
+	originalWorkDir, err := os.Getwd()
+	if err != nil {
+		t.Fatal(err)
+	}
+	// This builds and installs go-licenses CLI to $(go env GOPATH)/bin.
+	cmd := exec.Command("go", "install", ".")
```

Comment:
Let's `go build -o` into a tmp directory instead so we don't mutate a users copy of the tool.

Reply:
Good point! Fixed